src/Titon/Event/bootstrap.hh

```diff
@@ -15,6 +15,8 @@
 
 namespace Titon\Event {
     type CallStackList = Vector<string>;
+    type DataMap = Map<string, mixed>;
+    type EventList = array<Event>;
```

Comment:
On the edge about this one. I think we could go either way between arrays and vectors depending on the context. 

Would just be annoying to type this over and over.

```hack
$emitter->emitMany(Vector {new Event('foo'), new Event('bar')});
```

Instead of just.

```hack
$emitter->emitMany([new Event('foo'), new Event('bar')]);
```

I don't know though. Maybe we can audit use cases later on.